server/service/src/document/Programs.md

```diff
@@ -0,0 +1,144 @@
+# Programs module overview
+
+The program module supports doctors or practitioners to record medical patient data, to manage patients and to do patient related data reporting.
+A medical `program` focuses on the treatment of a medical condition and a patient can be enrolled in one or more programs.
+
+All program related data is recorded in `documents` which resemble physical forms.
+For example, there are currently three main document types:
+
+1. The `Patient` document records general patient information such as name, address or age.
+   The Patient document is shared between all programs of a patient.
+2. The `Program Enrolment` document holds patient specific information about a single program the patient is enrolled in.
+3. `Program Encounter` documents record data about a patients visit at a doctor.
```

Comment:
```suggestion
3. The `Program Encounter` documents record data about a patient's visit at a doctor.
```

Reply:
Changed to:
The `Program Encounter` document records data about a patient's visit at a doctor.